lib/commands/exam.js

```diff
@@ -48,6 +48,12 @@ module.exports = TestCommand.extend({
       description:
         'Randomizes your modules and tests while running your test suite.'
     },
+    {
+      name: 'module-path-filter',
+      type: String,
+      aliases: ['mpf'],
+      description: 'A module path to filter a module(s) to run.'
```

Comment:
Suggestion: `Filters the list of modules to only those that matches this value.`
Thoughts?

Reply:
What about A string to filter the list of modules to only those matches the string || the value?